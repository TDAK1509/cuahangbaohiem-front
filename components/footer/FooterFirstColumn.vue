<template>
  <div class="column">
    <figure class="image is-hidden-mobile mb-4">
      <img
        class="image footer-first-column__image"
        src="~assets/images/logo.png"
      />
    </figure>

    <ul class="footer-first-column__list is-size-7 ml-4">
      <li class="mt-1">Công ty cổ phần Cửa Hàng Bảo Hiểm</li>
      <li class="mt-1">
        Số 69, đường Lê Lai, phường Lê Lợi,<br />Thành phố Hồ Chí Minh
      </li>

      <li class="mt-1">
        Hotline: <a class="footer-first-column__link" href="tel:12345">12345</a>
      </li>
      <li class="mt-1">
        Email:
        <a
          class="footer-first-column__link"
          href="mailto:info@cuahangbaohiem.com"
          >info@cuahangbaohiem.com</a
        >
      </li>
    </ul>
  </div>
</template>

<script>
export default {
  name: "FooterFirstColumn"
};
</script>

<style lang="scss" scoped>
.footer-first-column__image {
  height: 40px;
  width: auto;
}

.footer-first-column__list {
  list-style-type: disc;
}

.footer-first-column__link {
  color: yellow;

  &:hover {
    color: #fff;
    text-decoration: underline;
  }
}
</style>
