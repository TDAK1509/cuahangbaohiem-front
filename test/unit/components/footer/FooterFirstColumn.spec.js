import { mount } from "@vue/test-utils";
import FooterFirstColumn from "@/components/footer/FooterFirstColumn.vue";

describe("FooterFirstColumn.vue", () => {
  it("matches snapshot", () => {
    const wrapper = mount(FooterFirstColumn);
    expect(wrapper.element).toMatchSnapshot();
  });
});
