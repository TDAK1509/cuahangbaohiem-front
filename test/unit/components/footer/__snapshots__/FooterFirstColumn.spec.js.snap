// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterFirstColumn.vue matches snapshot 1`] = `
<div
  class="column"
>
  <figure
    class="image is-hidden-mobile mb-4"
  >
    <img
      class="image footer-first-column__image"
      src="~assets/images/logo.png"
    />
  </figure>
   
  <ul
    class="footer-first-column__list is-size-7 ml-4"
  >
    <li
      class="mt-1"
    >
      Công ty cổ phần Cửa Hàng Bảo Hiểm
    </li>
     
    <li
      class="mt-1"
    >
      
      Số 69, đường Lê Lai, phường Lê Lợi,
      <br />
      Thành phố Hồ Chí Minh
    
    </li>
     
    <li
      class="mt-1"
    >
      
      Hotline: 
      <a
        class="footer-first-column__link"
        href="tel:12345"
      >
        12345
      </a>
    </li>
     
    <li
      class="mt-1"
    >
      
      Email:
      
      <a
        class="footer-first-column__link"
        href="mailto:info@cuahangbaohiem.com"
      >
        info@cuahangbaohiem.com
      </a>
    </li>
  </ul>
</div>
`;
